inbm/common-python-config/pyproject.toml

```diff
@@ -14,17 +14,17 @@ max_line_length = 99
 # complexity; see https://wemake-python-stylegui.de/en/latest/pages/usage/violations/complexity.html#complexity
 max_jones_score = 12
 max_cognitive_score = 208 # extremely high
-max_line_complexity = 21 # high
-max_imports = 47 # very high
+max_line_complexity = 24 # high
```

Comment:
max_imports increasing is probably fine, but max_line_complexity and max_expressions is good to keep lower if possible

Reply:
agree.  We need to get this back down.